projects/plugins/jetpack/extensions/blocks/one-payment/edit.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * External dependencies
+ */
+import { get, map } from 'lodash';
```

Comment:
There are some talks about deprecating lodash from Calypso because it's a large library, maybe we should try to avoid using lodash in Jetpack if it's possible?

Reply:
It's used elsewhere in Jetpack already, but its probably straight forward to drop it so sounds good to me.

Reply:
I tend to agree with @BogdanUngureanu even though I'm a heavy lodash user myself. 🙂 
Just be extra careful, because lodash functions do a lot of extra type checks that vanilla doesn't.

Given the fact that it's used everywhere in Jetpack, not using it here wouldn't change much, so consider first if it's actually worth it before replacing them.

Reply:
Lets just leave it for now then, there's plenty of more important stuff to be getting on with - lodash will already be loaded with the other eleventy uses of it in jp. 

I'll keep the advice in mind for the future though, and can always come back to this in 20% time (or just contribute to the more focused calypso effort)